tests/test_solvermode/test_speciestransport.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+
+@pytest.mark.quick
+@pytest.mark.setup
+@pytest.mark.fluent_version("latest")
+def test_species_material(load_mixing_elbow_mesh):
+    solver_session = load_mixing_elbow_mesh
+
+    # Test turning on species transport model
+    species_mdl = solver_session.setup.models.species.model
+
+    assert species_mdl.option.get_state() == "off"
```

Comment:
Do you have to call `.get_state()` here?  I would think this should work:
`assert species_mdl.option == "off"`

Reply:
What does `species_mdl.option` return? Compare it to that instead of the string `"off"`

Reply:
`species_mdl.option` is of type `ansys.fluent.core.solver.settings_241.option.option`. Calling `species_mdl.option()` returns a `str`.

Reply:
Right, thanks for the details. And does `species_mdl.option()` return the same as `species_mdl.option.get_state()`?

You are correct that using `get_state()` is more explicit, but in my understanding PyFluent users should generally not need these more API-backend calls such as `get_state`, `get_attr` and so on. In other words, we do not really want the more explicit commands here, we want to showcase the more simplified/easier-to-use interfaces if possible

Reply:
Yes, just calling the object itself returns the same thing. I'll replace the `get_state()` call with just `()`.